src/move/data/dataloaders.py

```diff
@@ -10,9 +10,23 @@
 from move.core.typing import BoolArray, FloatArray
 
 
-class MOVEDataset(TensorDataset):
-    "Characterizes a dataset for PyTorch"
+class MOVEDataset(TensorDataset):    
+    """
+    Characterizes a dataset for PyTorch
+
+    Attributes:
+        TensorDataset (class): class to represent data as list of tensors
+        
+    Args:
+        cat_all (torch.Tensor, optional): categorical input matrix (N_patients, N_variables x N_max-classes. Defaults to None.
+        con_all (torch.Tensor, optional): normalized continuous input matrix (N_patients, N_variables). Defaults to None.
+        cat_shapes (list, optional): list of tuples correspoding to number of features (N_variables, N_max-classes) of each categorical class. Defaults to None.
```

Comment:
Try to keep the line length to 80 characters.

Example:

```
        cat_all:
            Categorical input matrix (``num_samples`` x ``num_features`` x
            ``num_categories``). Defaults to None.
        con_all:
            Normalized continuous input matrix (``num_samples`` x 
            ``num_features``). Defaults to None.
```

Reply:
Fixed it: aa88bf3